tests/unit/core/test_tracking.js

```diff
@@ -353,4 +357,194 @@ describe(__filename, () => {
       sinon.assert.calledWith(fakeLog.log, 'Do Not Track is enabled');
     });
   });
+
+  describe('Hybrid Content Telemetry filterIdentifier()', () => {
+    it('should return content up to the default max length', () => {
+      expect(filterIdentifier('abcdefghijklmnopqrstuvwxyz').length).toEqual(20);
```

Comment:
You may want to use [`toHaveLength()`](https://jest-bot.github.io/jest/docs/expect.html#tohavelengthnumber) instead of `toEqual`.